.github/CODEOWNERS

```diff
@@ -5,48 +5,48 @@
-/pytorch_lightning/trainer/evaluation_loop.py  @tchaton @SeanNaren
-
-# Connectors
-/pytorch_lightning/trainer/connectors/         @tchaton @SeanNaren
-
-# accelerators
-/pytorch_lightning/accelerators/               @williamfalcon @tchaton @SeanNaren @awaelchli @justusschock
+/.github/CODEOWNERS @williamfalcon
+/README.md          @williamfalcon @edenlightning @borda
+/setup.py           @williamfalcon @borda
 
-# owners
-/.github/CODEOWNERS   @williamfalcon
-# main
-/README.md            @williamfalcon @edenlightning
-# installation
-/setup.py             @borda @williamfalcon
-
-# CI/CD
-/.github/workflows/   @borda @tchaton
-/.github/*.py         @borda @tchaton
-/dockers/             @borda @tchaton
-# configs in root
-/*.yml                @borda @tchaton
+# CI/CD and configs
+/.github/ @borda @tchaton
+/dockers/ @borda @tchaton
+*.yml     @borda @tchaton
 
 # Docs
-/docs/                          @edenlightning @tchaton @borda @awaelchli
-/.github/*.md                   @edenlightning @williamfalcon @borda
-/.github/ISSUE_TEMPLATE/*.md    @edenlightning @borda @tchaton
-/docs/source/conf.py            @borda @awaelchli
+/docs/                   @edenlightning @tchaton @borda @awaelchli
+/.github/*.md            @edenlightning @williamfalcon @borda
+/.github/ISSUE_TEMPLATE/ @edenlightning @borda @tchaton
+/docs/source/conf.py     @borda @awaelchli
+
+# Packages
+/pytorch_lightning/accelerators         @williamfalcon @tchaton @SeanNaren @awaelchli @justusschock
+/pytorch_lightning/callbacks            @williamfalcon @tchaton @carmocca @borda
+/pytorch_lightning/cluster_environments @borda @tchaton @SeanNaren @carmocca
+/pytorch_lightning/core                 @tchaton @SeanNaren @borda @carmocca
+/pytorch_lightning/distributed          @williamfalcon @tchaton @awaelchli
+/pytorch_lightning/loggers              @tchaton @awaelchli @borda
+/pytorch_lightning/overrides            @tchaton @SeanNaren @borda
+/pytorch_lightning/plugins              @tchaton @SeanNaren @awaelchli @justusschock
+/pytorch_lightning/profiler             @williamfalcon @tchaton @borda
+/pytorch_lightning/trainer              @williamfalcon @borda @tchaton @SeanNaren @carmocca
+/pytorch_lightning/trainer/connectors   @tchaton @SeanNaren @carmocca @borda
+/pytorch_lightning/tuner                @SkafteNicki @borda @awaelchli
+/pytorch_lightning/utilities            @borda @tchaton @SeanNaren @carmocca
+# Metrics
+/pytorch_lightning/metrics/             @teddykoker @ananyahjha93 @justusschock
+/tests/metrics/                         @teddykoker @ananyahjha93 @justusschock
+/docs/source/metrics.rst                @teddykoker @ananyahjha93 @justusschock
+
+# API
+/pytorch_lightning/callbacks/base.py  @williamfalcon @tchaton
+/pytorch_lightning/core/datamodule.py @williamfalcon @tchaton
+/pytorch_lightning/trainer/trainer.py @williamfalcon @tchaton
+/pytorch_lightning/core/hooks.py      @williamfalcon @tchaton
+/pytorch_lightning/core/lightning.py  @williamfalcon @tchaton
```

Comment:
can you guys add more (maybe 1 more) here??

Reply:
I'm not sure if @williamFalcon would be okay with that

Reply:
I think, that's exactly what @tchaton requested above?

Reply:
if only william and thomas are listed, will we still get request for review?

Reply:
Not for these files

Reply:
that's unfortunate :(

Reply:
Let's add @SeanNaren there and see if Will accepts this update.

Reply:
@awaelchli we can add you and me as a reviewer by merfigy even we won't be code-owners... :]

Reply:
@awaelchli have look at https://docs.mergify.io/conditions.html#how-to-match-lists
```
files=README is true if the file README is modified in the pull request.
files!=README is true if the file README is not modified in the pull request.
files~=^src/ is true if any files in the src directory is modified in the pull request.
-files~=^src/ is true if none of the files that are modified are in the src directory.
```